src/commands/config.js

```diff
@@ -365,12 +385,36 @@ exports.handler = async ({func} = {}) => {
     }
   };
 
-  // get values from user
-  const newConfig = await inquirer.prompt(prompts);
+  let newConfig = defaultConfig;
+  let nonInteractive = false;
```

Comment:
I'd generate the value in-place instead of inside a function.
The easiest way would be to do something like this in:
```js
// in function definition - separate func and non-interactive args
exports.handler = async ({func, ...args} = {}) => {
  // rest of the logic...

  const nonInteractive = Object.keys(args).some(key => args[key].length > 0);

  // reset of the logic ...
}
```

Reply:
I dont get the stream logic not right...
There is also a parameter `func` which is not eligable for "non interactive" :(